<launch>
<include file="$(find finder_gazebo)/launch/finder_apartment.launch" />
<include file="$(find finder_bringup)/launch/finder_gmapping.launch" />
<include file="$(find exploration)/launch/Exploration_Services.launch" />
<node pkg="exploration" type="autonomous_exploration_client.py" name="autonomous_exploration_client" respawn="false" output="screen"></node>
</launch>
